src/KeyVault/KeyVault/Models/KeyVaultCmdletBase.cs

```diff
@@ -68,5 +70,38 @@ protected void GetAndWriteObjects<TObject>(KeyVaultObjectFilterOptions options,
                 WriteObject(pageResults, true);
             } while (!string.IsNullOrEmpty(options.NextLink));
         }
+        
+        public List<T> KVSubResourceWildcardFilter<T>(string name, IEnumerable<T> resources)
```

Comment:
@maddieclayton is there any reason that we need to implement this method and can't just use the following:

```cs
TopLevelWildcardFilter(null, name, resources)
```